atlp-rwanda/e-comm-team-brogrammers-bn
# #184581321 Integrate coveralls code coverage service

#### What does this PR do?
* Integrate coveralls code coverage service

#### Description of Task to be completed?
* This is to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product
#### How should this be manually tested?
* use the command "npm run test",  and you will be able to see all the coverage and coveralls
#### Any background context you want to provide?
* Coveralls is an automated code coverage service that integrates with GitHub to provide insights into code coverage and quality. It provides a variety of features such as real-time tracking of code coverage, showing which parts of the code are tested and which are not, and helping to identify areas where bugs may be lurking. It also offers a range of integrations with other services, making it easy to integrate with existing workflows.

#### What are the relevant pivotal tracker/Trello stories?
*[#184581321] ( https://www.pivotaltracker.com/n/projects/2632830/stories/184581321)
#### Screenshots (if appropriate)

#### Questions:

* What are the steps required to ensure that all existing code is covered by the code coverage service?

* What percentage of code coverage is acceptable for our project?

* What is the easiest way to integrate Coveralls with my project?

* What is the process for integrating Coveralls with an existing automated testing and deployment process?
